numba/cuda/cudadrv/nvvm.py

```diff
@@ -429,6 +429,46 @@ def get(self):
 }}
 """ # noqa: E501
 
+ir_numba_atomic_inc = """
```

Comment:
I think `ir_numba_atomic_inc` and `ir_numba_atomic_dec` need no longer be templated now they're only used for `i64`, but I'm also happy with them as they are if you'd prefer to leave it this way.

Reply:
From out-of-band conversation, this will be left as-is.